.eslintrc.js

```diff
@@ -12,7 +12,8 @@ module.exports = {
     'airbnb',
     'plugin:jsx-a11y/recommended',
     'prettier',
-    'plugin:grommet/recommended',
+    // TO DO resolve why definitions weren't found
```

Comment:
Should we create a separate issue to look into this?

Reply:
Yes: https://github.com/grommet/hpe-design-system/issues/3403